jessie_update/Raspbian_for_Robots_Flavor.sh

```diff
@@ -75,9 +77,10 @@ echo "Hostname change will be effective after a reboot"
 # many many thanks to Russell Davis for all the hints!
 # tightvncserver will only work after a reboot - not done here
 ####################################
+echo "********** TIGHTVNCSERVER ***********"
 sudo wget https://raw.githubusercontent.com/DexterInd/Raspbian_For_Robots/master/jessie_update/tightvncserver.sh
-sudo chmod +x tightvncserver.sh
-./tightvncserver.sh $DEFAULT_PWD
+#sudo chmod +x tightvncserver.sh
```

Comment:
Are we removing this because of using BASH instead of directly executing this from the terminal